jdaviz/app.py

```diff
@@ -327,6 +327,9 @@ def load_data(self, file_obj, parser_reference=None, **kwargs):
             snackbar_message = SnackbarMessage("Data successfully loaded.",
                                                sender=self)
             self.hub.broadcast(snackbar_message)
+        except Exception:  # Reset state on uncaught errors
+            self.data_collection.clear()
```

Comment:
So if I load data 2 separate times and this fails on the second time, will ALL data be removed from data_collection?

Reply:
Good point... I'll have to ponder a bit more.

Reply:
I think I addressed it in second commit. Thanks for pointing that out!